client/src/components/ProfileDetailPanel.js

```diff
@@ -6,7 +6,9 @@ const useStyles = makeStyles((theme) => ({
   paper: {
     padding: theme.spacing(2),
     textAlign: 'center',
-    width: '30%',
+    '@media (min-width: 990px)': {
+        width: '30%',
+    },
```

Comment:
Top width should be mobile width and @media width should be desktop, atm it looks not that different.

also replace the @media with `[theme.breakpoints.up('md')]: {...}` See NavBar.js in components for example

Reply:
fixed the @media, but unclear on what you mean by top width. I was using the default for mobile and the query for desktop.

Reply:
sorry I'll fix it in my slack chat with help, but the media query is not doing anything because they are the same value